spec/values/calculation/sqrt.hrx

```diff
@@ -0,0 +1,146 @@
+<===> README.md
+Most of the same behavior tested for `calc()` applies to `sqrt()`, but for
+terseness' sake isn't tested explicitly.
+
+<===>
+================================================================================
+<===> zero/input.scss
+a {b: sqrt(0)}
+
+<===> zero/output.css
+a {
+  b: 0;
+}
+
+<===>
+================================================================================
+<===> result_is_integer/input.scss
```

Comment:
You don't need separate integer/float tests

Reply:
For `sqrt` specifically I thought it might be useful, but now that I think about it it's the same since Sass treats all numbers as floats.